src/pybind/mgr/prometheus/module.py

```diff
@@ -1030,6 +1133,16 @@ def get_metadata_and_osd_status(self) -> None:
                 self.log.info("Missing dev node metadata for osd {0}, skipping "
                               "occupation record for this osd".format(id_))
 
+        if 'disk_occupation' in self.metrics:
+            try:
+                self.metrics['disk_occupation_human'] = \
+                    self.metrics['disk_occupation'].group_by(
+                        ['device', 'instance'],
+                        {'ceph_daemon': lambda daemons: ', '.join(daemons)},
```

Comment:
```suggestion
                        {'ceph_daemon': lambda daemons: ', '.join(daemons)},
                        name='disk_occupation_human'
```
With this change then `ceph_disk_occupation_human` metric is generated in my local environment.